src/lib/y2storage/resize_info.rb

```diff
@@ -32,8 +33,33 @@ module Y2Storage
   # This is a wrapper for Storage::ResizeInfo
   class ResizeInfo
     include StorageClassWrapper
+    include Yast::I18n
+    extend Yast::I18n
     wrap_class Storage::ResizeInfo
 
+    # rubocop:disable Metrics/LineLength
+    REASON_TEXTS =
+      {
+        RB_RESIZE_NOT_SUPPORTED_BY_DEVICE:       N_("Resizing is not supported by this device."),
+        RB_MIN_MAX_ERROR:                        N_("Combined limitations of partition and filesystem prevent resizing."),
+        RB_SHRINK_NOT_SUPPORTED_BY_FILESYSTEM:   N_("This filesystem does not support shrinking."),
+        RB_GROW_NOT_SUPPORTED_BY_FILESYSTEM:     N_("This filesystem does not support growing."),
+        RB_FILESYSTEM_INCONSISTENT:              N_("Filesystem consistency check failed."),
+        RB_MIN_SIZE_FOR_FILESYSTEM:              N_("This filesystem already has the minimum possible size."),
+        RB_MAX_SIZE_FOR_FILESYSTEM:              N_("This filesystem already has the maximum possible size."),
+        RB_FILESYSTEM_FULL:                      N_("The filesystem is full."),
```

Comment:
Maybe extend it to "The filesystem is full. The shring is not possible. If you need shrink, consider freeing some space there."

Reply:
At that point, it isn't even clear that this was about shrinking. All we get from the library are the flags which are converted to those Ruby symbols (looking very much like constants) here. It may be the common thing, but it's not necessarily only shrinking that is prevented here; from all we know at this point, it might be any operation.